test/node/jsdom.js

```diff
@@ -0,0 +1,54 @@
+var axe = require('../../');
+var jsdom = require('jsdom');
```

Comment:
Since we can use "modern JS" here, I really think we should. Thoughts on refactoring this to make it look like "today's JavaScript"?

Reply:
I'd love to but we can't. The entire `test` dir is dictated by the [eslint file](https://github.com/dequelabs/axe-core/blob/develop/test/.eslintrc) in there that disallows any modern javascript... It's a bit of a pain.

Reply:
That doesn't sound like a "we can't", it just sounds like some ESLint settings need to be modified. We could add `test/node/.eslintrc` and override the rules added there, or move that file into the directory(ies) that those rules are relevant for.

I don't consider this blocking tho. If it's too much work to mess with the ESLint stuff, I understand. We can always improve on this later.